extra/installer.nsi

```diff
@@ -0,0 +1,115 @@
+;-------------------------------------------------------------------------------
+; Includes
+!include "MUI2.nsh"
+!include "LogicLib.nsh"
+!include "WinVer.nsh"
+!include "x64.nsh"
+
+;-------------------------------------------------------------------------------
+; Constants
+!ifndef PRODUCT_NAME
+    !define PRODUCT_NAME "Refl1D"
+!endif
+!define PRODUCT_DESCRIPTION "Bayesian Uncertainty Modeling for the Physical Sciences"
+!define COPYRIGHT "Copyright © 2018 The Refl1D developers"
+!define PRODUCT_VERSION 1.0.0.1013
```

Comment:
Refl1d release version and copyright should be supplied on the command line instead of hard coding them here.

Reply:
True.  If we can just leave them out here that would be better.

Reply:
It looks like this info does show up if you right click on the installer and view Properties -> Details.  It won't be too much effort to add the correct information.